### DISCUSSION TITLE
Create custom login page with default styled buttons

### DISCUSSION THREAD

**Discussion by JuliusMoehring** [nextauthjs/next-auth#7352] [Help]
Hi, 

I am using NextAuth for the first time and I love it so far. There is one thing where I didn't find any documentation about: 

Is it possible to create an custom login page, but use the already styled buttons that you normally get without the custom login page?

Thanks for the help!

**Comment by Austin-Brinegar** [comment]
Hello, I'm having this same issue. Were you ever able to find a solution?
